### ISSUE TITLE
import does not work any more

### ISSUE THREAD

**Issue by ulrichSchreiner** [davidhu2000/react-spinners#66]
hi,

i just updated from version 0.5.4 to 0.5.7 and now my import does not work any more. i used to do
~~~
import { GridLoader } from 'react-spinners';
~~~
but this returns a undefined. i changed my code to
~~~
import GridLoader from 'react-spinners/GridLoader';
~~~
which works. 

perhaps the commit 0a08529a8eb9f141d92c07c2a208e5e1ba253891 has a negative side effect?

**Comment by taweesoft** [comment]
Same here. I was figuring it out for hours. 0.5.7

**Comment by benkeen** [comment]
Thanks for posting the workaround @ulrichSchreiner, same here.

**Comment by bytewoopy** [comment]
Same issue, had to rollback to previous version.

**Comment by davidhu2000** [comment]
Thank for opening this issue. I'll look into this issue. https://github.com/davidhu2000/react-spinners/commit/0a08529a8eb9f141d92c07c2a208e5e1ba253891 is the cause, but that change is needed in order to not break the demo site. For now, i'd keep using `0.5.4` because the new patch versions do not contain any new features, just devDependency updates

**Comment by davidhu2000** [comment]
This should now be fixed on `0.5.8`. Had to update `export default` to `export const` for each individual loader. https://github.com/davidhu2000/react-spinners/blob/master/src/index.js

Although I'd recommend importing the loaders via 

```
import GridLoader from 'react-spinners/GridLoader';
```

to help reduce import cost, since this would only import the loader you need.

**Comment by davidhu2000** [comment]
going this issue now. Please let me know if this issue continues.
